rpc/json/handler.go

```diff
@@ -23,15 +24,17 @@ type handler struct {
 	mux    *http.ServeMux
 	codec  rpc.Codec
 	logger log.Logger
+	config *tmconfig.RPCConfig
```

Comment:
Actually, we already have [`RPCConfig` type](https://github.com/rollkit/rollkit/blob/ea0dec6801c84db7333b24f9d5c022ccc68557ba/config/rpc.go#L4) to avoid using tendermint types directly deep in rollkit.
There's also potential to extract a method from: https://github.com/rollkit/rollkit/blob/ea0dec6801c84db7333b24f9d5c022ccc68557ba/conv/config.go#L21-L29).